kubernetes/polkassembly/front-end-deployment.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  annotations:
+  labels:
+    app: front-end
+  name: front-end
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: front-end
+  template:
+    metadata:
+      labels:
+        app: front-end
+    spec:
+      containers:
+      - name: front-end
+        image: gcr.io/test-installations-222013/polkassembly-front-end:master
+        env:
+        - name: REACT_APP_AUTH_SERVER_GRAPHQL_URL
+          value: "http://auth-server-service:8010/auth/graphql"
+        - name: REACT_APP_ENV
+          value: "dev"
+        - name: REACT_APP_HASURA_GRAPHQL_URL
+          value: "http://hasura-service:8080/v1/graphql"
+        - name: REACT_APP_JWT_PUBLIC_KEY
+          value: "-----BEGIN PUBLIC KEY-----\nMIIBI...wIDAQAB\n-----END PUBLIC KEY-----"
```

Comment:
This will not work. Full public key is needed here.

Reply:
-----BEGIN PUBLIC KEY-----
MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAohj4C7n/5PzCANMukmXF
wE/wGK9vBRnaWZCODu5zg9mcVrgiuFlubONSoVg9DSCxSlPmaKjAZUy1xARNKPgL
caEkxteT6d5p50iiL0iMTtz85Y5yefoQAWFNjgN+VbiVEycWvw5uKYOrQkUIuxYm
jcI5xsmDU48UTha1HFgC6r+It9ViZrsfTAwj4z9lvOcPphVLh9TFaON8LnWPisly
F4r6uPY6nLdDVvbDRlWoAEeg7mXaxBFWZidi9pKBMFavxyHhH3iyg4dCvxKzEH+X
RA9i6Zw1c02Jgyd6viBAkB963SrGTAF+TNzepidoK5B/sIZD62OseoMX8fBXEu5e
mQIDAQAB
-----END PUBLIC KEY-----